.circleci/config.yml

```diff
@@ -610,18 +622,30 @@ jobs:
             - install_helm
             - run:
                 command: |
-                    LATEST_TAG_WITH_V=`git describe --abbrev=0 --tags ${CIRCLE_SHA1}` &&
-                    LATEST_TAG=${LATEST_TAG_WITH_V:1} &&
-                    IMAGE_NAME_APPROVED=snyk/kubernetes-monitor:${LATEST_TAG}-approved &&
-                    IMAGE_NAME_PUBLISHED=snyk/kubernetes-monitor:${LATEST_TAG} &&
+                    LATEST_TAG_WITH_V=`git describe --abbrev=0 --tags ${CIRCLE_SHA1}`
+                    LATEST_TAG=${LATEST_TAG_WITH_V:1}
+                    IMAGE_NAME_APPROVED=snyk/kubernetes-monitor:${LATEST_TAG}-approved
+                    IMAGE_NAME_PUBLISHED=snyk/kubernetes-monitor:${LATEST_TAG}
+                    echo "export LATEST_TAG=${LATEST_TAG}" >> $BASH_ENV
+                    echo "export IMAGE_NAME_APPROVED=${IMAGE_NAME_APPROVED}" >> $BASH_ENV
+                    echo "export IMAGE_NAME_PUBLISHED=${IMAGE_NAME_PUBLISHED}" >> $BASH_ENV
+                name: Export environment variables
+            - snyk/scan:
```

Comment:
why is this duplicated below? L633-635 is same as L636, 638, 639

Reply:
The first scan is `snyk test` which is testing package.json, the second scan is `snyk container test` which tests the image with the Dockerfile

Reply:
oh I see! confusing they are called the same thing